### ISSUE TITLE
azuredevops_git_repository init clean creates a lower case readme.md

### ISSUE THREAD

**Issue by slzmruepp** [microsoft/terraform-provider-azuredevops#695]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Affected Resource(s)

* `azuredevops_git_repository`

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Expected Behavior

If I create a clean Repo with initializing it, it should follow the Azure DevOps initializing approach which creates a README.md in upper case.

### Actual Behavior

Initialization init_type = "Clean" creates a lower case "readme.md"

### Steps to Reproduce

1. `terraform apply`

### References

* #0000

**Comment by xuzhang3** [comment]
@slzmruepp File name should be the same as the input name. Can you share your tf config?

**Comment by christopherfriedrich** [comment]
We have the same problem. I had a quick look into the code, when creating a repository of init type `clean` it actually creates one commit putting this `readme.md` file into the repository: https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/service/git/resource_git_repository.go#L296-L312.

**Comment by slzmruepp** [comment]
@xuzhang3 Its hardcoded in the provided code sample.

**Comment by xuzhang3** [comment]
@slzmruepp @christopherfriedrich  you are right. `init_type` was introduced to initial the repository and was hardcode. We are considering replace the `init_type` with Terraform resources like `azuredevops_git_repository_file`.

**Comment by fernandosalomao** [comment]
> @slzmruepp @christopherfriedrich you are right. `init_type` was introduced to initial the repository and was hardcode. We are considering replace the `init_type` with Terraform resources like `azuredevops_git_repository_file`.

Good idea. It might be useful to support multiple files as well > https://github.com/microsoft/terraform-provider-azuredevops/issues/696#issuecomment-1529282066

**Comment by fernandosalomao** [comment]
> > @slzmruepp @christopherfriedrich you are right. `init_type` was introduced to initial the repository and was hardcode. We are considering replace the `init_type` with Terraform resources like `azuredevops_git_repository_file`.
> 
> Good idea. It might be useful to support multiple files as well > [#696 (comment)](https://github.com/microsoft/terraform-provider-azuredevops/issues/696#issuecomment-1529282066)

Meanwhile I have submitted a PR to uppercase it.

**Comment by xuzhang3** [comment]
Fixed by #761
